ecmascript/transforms/module/src/common_js.rs

```diff
@@ -699,6 +704,15 @@ where
                     stmts.push(require.into_stmt().into());
                 }
             }
+
+            let exported = export_alls.remove(&src);
+            if let Some(export) = exported {
+                stmts.push(ModuleItem::Stmt(Scope::handle_export_all(
```

Comment:
Curious if this approach makes sense? it was not possible to call `handle_export_all` due to move occurrence, I took out part to create Ident (which required &mut self) hold by hashmap, and actual stmt creation occurs in here.

Reply:
Yes, I think it's correct approach.